Troubleshooting added to check available components in the collector

nerudadhich opened the pull request:
Fix for issue - https://github.com/open-telemetry/opentelemetry.io/issues/2072

tiffany76 commented on content/en/docs/collector/troubleshooting.md:
```diff
@@ -15,6 +15,172 @@ network issues, it can be helpful to send a small amount of data to a collector
 configured to output to local logs. For details, see
 [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md#local-exporters).
 
+## Check available components in the collector
```
```suggestion
## Check available components in the Collector
```

svrnm replied:
@theletterf do I remember correctly that we agreed that we write "Collector" if we are referring to the OpenTelemetry Collector?

tiffany76 reviewed (commented):
I've made a few small suggestions to help with clarity and conform to the style guide. 

The sample output seems a little long to me. Maybe showing only a couple of each type of component would be clearer?  The Comms folks will know best.

nerudadhich commented:
> I've made a few small suggestions to help with clarity and conform to the style guide.
> 
> The sample output seems a little long to me. Maybe showing only a couple of each type of component would be clearer? The Comms folks will know best.

Thanks for comment, I agree with sample output. Let's wait for more inputs.

mx-psi reviewed (commented):
Should we note that the output format may change across versions?

nerudadhich commented:
> Should we note that the output format may change across versions?

I have updated the sample output and added the note as well. Please review again and confirm if anything else need to add/update.

tiffany76 reviewed (commented):
LGTM!
